manual/src/cmds/native.etex

```diff
@@ -182,6 +182,14 @@ the following environment variables are also consulted:
   "CAMLRUNPARAM" is not found, then the default values will be used.
 \end{options}
 
+\begin{windows}
+On Windows, an "ocamlopt"-generated executable will expand the "*" and "?"
+wildcards in arguments before anything else. The string array "Sys.argv" is
+constructed using the expanded arguments. If such a behavior were to be avoided,
+it is possible to write a custom entry point in C.
```

Comment:
I think "were to be avoided" does not mean exactly what we want in English; suggestion below:
```suggestion
constructed using the expanded arguments. To avoid this behaviour, it is necessary
to write a custom entry point in C.
```
Also, I would add a reference to the section in the manual that explains how to write a custom entry point in C (https://ocaml.org/manual/5.2/intfc.html#ss:main-c).

Reply:
Fixed, thanks for spotting this!